e2e/my-posts.spec.ts

```diff
@@ -1,6 +1,27 @@
+import type { Page } from "@playwright/test";
 import test, { expect } from "@playwright/test";
 import { articleExcerpt, loggedInAsUserOne } from "./utils";
 
+async function openPublishedTab(page: Page) {
+  await page.goto("http://localhost:3000/my-posts");
+  await page.getByRole("link", { name: "Published" }).click();
+  await expect(page).toHaveURL(/\/my-posts\?tab=published$/);
+}
+
+async function openDeleteModal(page: Page) {
+  await page.getByRole("button", { name: "Options" }).click();
+  const optionsDiv = page.locator(
+    "div[aria-labelledby='headlessui-menu-button-:r5:']",
```

Comment:
nitpick: can we do a little readable locator instead of this headless ui dependent one

Reply:
That is updated @pkspyder007 and looks much cleaner, thanks for the tip.